crates/rust-analyzer/src/handlers.rs

```diff
@@ -2,6 +2,8 @@
 //! Protocol. The majority of requests are fulfilled by calling into the
 //! `ra_ide` crate.
 
+#![allow(deprecated)]
```

Comment:
Why do we need this? Let's put it on a specific expression

Reply:
The `deprecated` field is deprecated in favor of `tags`.